hardware/asic/README.md

```diff
@@ -62,9 +61,16 @@ sudo usermod -aG docker $USER
 ```bash
 newgrp docker
 ```
-Now you should be able to mount docker without sudo access
+Now you should be able to mount docker without sudo access.
 
-This command opens up a bash terminal in which you run the following tcl script to generate a design configuration of your design:
+## OPENLANE FLOW FOR MACRO HARDENING FROM AN HDL DESIGN
+
+1. Go to the OpenLane installation directory (you may set an environment variable called OPENLANE_HOME that points to this directory) and start the OpenLane Docker container by running 
+
+```bash
+make mount
+```
+The "make mount" command opens up a bash terminal in which you run the following tcl script to generate a design configuration of your design:
```

Comment:
In which line in the README the user is instructed to create the design files? Where should the user put them?

Reply:
I have provided the required explanation.

Reply:
Kindly check changes